docs/testing/debug-with-vs-extension.md

```diff
@@ -1,84 +1,119 @@
 
-## Install and use the debugger
+1. The first step depends on the project and how it was created.
 
-1. If you need to create an add-in project, [use the Yo Office generator to create one](../quickstarts/excel-quickstart-jquery.md?tabs=yeomangenerator). Follow the prompts within the command line to set up your project. You can choose any language or type of project to suit your needs. This tutorial uses an Excel task pane add-in.
+   - If you want to create a project to experiment with debugging in Visual Studio Code, use the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office). You can use any one of our quick start guides, such as the [Outlook add-in quickstart](../quickstarts/outlook-quickstart.md), in order to do this. 
+   - If you want to debug an existing project that was created with Yo Office, skip to the next step.
+   - If you want to debug an existing project that was not created with Yo Office, carry out the procedure in the [Appendix](#appendix) and then return to the next step of this procedure.
 
-    > [!NOTE]
-    > If you already have a project, skip step 1 and move to step 2.
 
-1. Open a command prompt as administrator.
-   ![Command prompt options, including "run as administrator" in Windows 10 and 11.](../images/run-as-administrator-vs-code.jpg)
+1. Open VS Code and open your project in it. 
 
-1. Navigate to your project directory.
+1. Within VS Code, select **Ctrl+Shift+X** to open the Extensions bar. Search for the "Microsoft Office Add-in Debugger" extension and install it.
 
-1. Run the following command to open your project in Visual Studio Code as an administrator.
+1. hoose  **View > Run** or enter **Ctrl+Shift+D** to switch to debug view.
 
-    ```command&nbsp;line
-    code .
-    ```
+1. From the **RUN AND DEBUG** options, choose the Edge Legacy option for your host application, such as **Excel Desktop (Edge Legacy)**. Select **F5** or choose **Run > Start Debugging** from the menu to begin debugging. This action automatically launches a local server in a Node window to host your add-in and then automatically opens the host application, such as Excel or Word. This may take several seconds.
 
-  Once Visual Studio Code is open, navigate manually to the project folder.
+1. In the host application, your add-in is now ready to use. Select **Show Taskpane** or run any other add-in command. A dialog box will appear, reading:
 
-  > [!TIP]
-  > To open Visual Studio Code as an administrator, select the **run as administrator** option when opening Visual Studio Code after searching for it in Windows.
+   > WebView Stop On Load.
+   > To debug the webview, attach VS Code to the webview instance using the Microsoft Debugger for Edge extension, and click OK to continue. To prevent this dialog from appearing in the future, click Cancel.
```

Comment:
```suggestion
   > To debug the WebView, attach VS Code to the WebView instance using the Microsoft Debugger for Edge extension, and click **OK** to continue. To prevent this dialog from appearing in the future, click **Cancel**.
```